src/lexer.coffee

```diff
@@ -473,7 +476,11 @@ exports.Lexer = class Lexer
         @error message, origin[2] if message
       return value.length if skipToken
 
-    if value is '{' and prev?[0] is 'EXPORT'
+    if value is '{' and @seenImport
+      @importSpecifierList = yes
+    else if @importSpecifierList and value is '}'
```

Comment:
@vendethiel right, I didn't see any cases in the documentation or tests where an `import` could contain nested-object syntax?